include/exec/sequence_senders.hpp

```diff
@@ -0,0 +1,777 @@
+
+  inline constexpr __enumerate_each::enumerate_each_t enumerate_each{};
+
+  namespace __join_all {
+    using namespace stdexec;
+
+    template <class _ReceiverId>
+    struct __receiver {
+      using _Receiver = stdexec::__t<_ReceiverId>;
+
+      struct __t {
+        [[no_unique_address]] _Receiver __rcvr_;
+
+        template <class _Item>
+        friend _Item&& tag_invoke(set_next_t, __t&, _Item&& __item) noexcept {
+          return static_cast<_Item&&>(__item);
+        }
+
+        template <class _Error>
+        friend void tag_invoke(set_error_t, __t&& __self, _Error&& __error) noexcept {
+          set_error(static_cast<_Receiver&&>(__self.__rcvr_), static_cast<_Error&&>(__error));
+        }
+
+        template <__one_of<set_stopped_t, set_value_t> _Tag>
+        friend void tag_invoke(_Tag complete, __t&& __self) noexcept {
+          complete(static_cast<_Receiver&&>(__self.__rcvr_));
+        }
+
+        friend env_of_t<_Receiver> tag_invoke(get_env_t, const __t& __self) noexcept {
+          return get_env(__self.__rcvr_);
+        }
+      };
+    };
+
+    template <class _Rcvr>
+    using __receiver_t = __t<__receiver<__id<__decay_t<_Rcvr>>>>;
+
+    template <class... _Args>
+    using __drop_value_args = completion_signatures<set_value_t()>;
+
+    template <class _Sender, class _Env>
+    using __completion_sigs = make_completion_signatures<
+      _Sender,
+      _Env,
+      completion_signatures<set_value_t()>,
+      __drop_value_args>;
+
+    template <class _SenderId>
+    struct __sender {
+      using _Sender = stdexec::__t<__decay_t<_SenderId>>;
+
+      struct __t {
+        [[no_unique_address]] _Sender __sndr_;
+
+        template <__decays_to<__t> _Self, class _Receiver>
+          requires receiver_of<
+                     _Receiver,
+                     __completion_sigs<__copy_cvref_t<_Self, _Sender>, env_of_t<_Receiver>>>
+                && sequence_sender_to<__copy_cvref_t<_Self, _Sender>, __receiver_t<_Receiver>>
+        friend sequence_connect_result_t< __copy_cvref_t<_Self, _Sender>, __receiver_t<_Receiver>>
+          tag_invoke(connect_t, _Self&& __self, _Receiver&& __rcvr) {
+          return sequence_connect(
+            ((_Self&&) __self).__sndr_, __receiver_t<_Receiver>{static_cast<_Receiver&&>(__rcvr)});
+        }
+
+        template <__decays_to<__t> _Self, class _Env>
+        friend auto tag_invoke(get_completion_signatures_t, _Self&&, const _Env&)
+          -> __completion_sigs<__copy_cvref_t<_Self, _Sender>, _Env>;
+      };
+    };
+
+    struct join_all_t {
+      template <class _Sender>
+      constexpr auto operator()(_Sender&& __sndr) const {
+        return __t<__sender<__id<__decay_t<_Sender>>>>{static_cast<_Sender&&>(__sndr)};
+      }
+
+      constexpr auto operator()() const noexcept -> __binder_back<join_all_t> {
+        return {};
+      }
+    };
+  } // namespace __join_all
+
+  using __join_all::join_all_t;
+  inline constexpr join_all_t join_all;
```

Comment:
FYI I called this ignore_all. it does not join in the async sense..

Reply:
Yup, that's better, thank you! I didn't know how to name that